src/main/java/com/zemnov/salon/model/User.java

```diff
@@ -14,18 +14,32 @@
 @Entity
 @Table(name="usr")
 public class User implements UserDetails {
+
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
-    private Integer id;
+    private Long id;
     private String username;
     private String password;
+    private String clientName;
+    private String number;
+    private String mail;
+
+    @OneToOne(fetch = FetchType.EAGER, cascade = {CascadeType.REMOVE})
+    @JoinColumn(name = "contact_id")
+    private Contact contact;
     private boolean active;
 
     @ElementCollection(targetClass = Role.class, fetch = FetchType.EAGER)
     @CollectionTable(name = "user_role", joinColumns = @JoinColumn(name = "user_id"))
     @Enumerated(EnumType.STRING)
     private Set<Role> roles;
 
+    public boolean isAdmin() {
```

Comment:
It should not be placed here, it is out of this class responsibilities. Let's create separate interface and implementaion for such logic

Reply:
I use this method to manage security in freemarker parts. Not sure what exactly should I do and how to sepparate this method

Reply:
I suppose, we can just add 'isAdmin' boolean field, and set it during the object creating